es/lib/isEmail.js

```diff
@@ -1,8 +1,12 @@
-function _slicedToArray(arr, i) { return _arrayWithHoles(arr) || _iterableToArrayLimit(arr, i) || _nonIterableRest(); }
+function _slicedToArray(arr, i) { return _arrayWithHoles(arr) || _iterableToArrayLimit(arr, i) || _unsupportedIterableToArray(arr, i) || _nonIterableRest(); }
```

Comment:
Remove this file from the diff since it's unrelated to the PR.

Reply:
this was auto-generated

Reply:
Just remove it before committing. This is because of the difference in the build system.